assets/js/modules/ads/components/notifications/AdsModuleSetupCTABanner.js

```diff
@@ -154,6 +154,16 @@ export default function AdsModuleSetupCTABanner( { id, Notification } ) {
 	const showTooltip = useShowTooltip( id );
 	const { isTooltipVisible } = useTooltipState( id );
 
+	const [ dismissLabel, setDismissLabel ] = useState(
```

Comment:
This was required so that the text within the secondary (dismissal) CTA did not change prior to the redirect.